modules/gateways/Blockonomics/Blockonomics.php

```diff
@@ -204,9 +245,11 @@ public function getNewBitcoinAddress($reset=false) {
 
 		$ch = curl_init();
 
-		curl_setopt($ch, CURLOPT_URL, "https://www.blockonomics.co/api/new_address" . $get_params);
+		curl_setopt($ch, CURLOPT_URL, "https://".$subdomain.".blockonomics.co/api/new_address" . $get_params);
 		curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
 		curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "POST");
+		curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 20); 
+		curl_setopt($ch, CURLOPT_TIMEOUT, 20); //timeout in seconds
```

Comment:
Twenty seconds timeout is very large! Anyway we haven't seen any address generation timeout issues on WHMCS. Let us remove this

Reply:
Removed in 97b9f5c